css/global.css

```diff
@@ -10,4 +10,8 @@
 
 .c-message_kit__hidden_message_blur {
   filter: blur(0) !important;
+}
+
+.c-message_kit__hidden_message_blur * {
+  pointer-events: auto !important;
```

Comment:
And here it's overriding the `pointer-events` property value to be the default value, `auto`, which is the same as not having a `pointer-events` property and value at all.